src/views/webview-app/extension-app-message-constants.ts

```diff
@@ -123,10 +137,12 @@ export type MESSAGE_FROM_WEBVIEW_TO_EXTENSION =
   | OpenConnectionStringInputMessage
   | OpenTrustedLinkMessage
   | RenameConnectionMessage
-  | EditAndConnectConnection;
+  | EditAndConnectConnection
```

Comment:
nit: maybe `EditConnectionAndConnect`?

Reply:
Yeah, this is from before, but I agree I could have a better name :) I will update!